### ISSUE TITLE
Package name does not follow ros convention

### ISSUE THREAD

**Issue by nullket** [Project-MANAS/ars_40X#4]
When building and launching one does get a warning that the package does not follow the ros (naming convention)[http://wiki.ros.org/ROS/Patterns/Conventions]:
```
WARNING: Package name "ars_40X" does not follow the naming conventions. It should start with a lower case letter and only contain lower case letters, digits, underscores, and dashes.
```
I would suggest changing the name `ars_40X` to `ars_40x`. What do you think?

**Comment by LukeAI** [comment]
Agreed, every time I do build or launch anything in the same workspace as this package I get a bunch of annoying warnings like this. I am for a warning-free workspace and I see this all day every day. Would be a relatively big improvement for a tiny effort. @shiveshkhaitan  if I put in the PR will you accept it?
